pl_examples/domain_templates/computer_vision_fine_tuning.py

```diff
@@ -166,23 +163,24 @@ class TransferLearningModel(pl.LightningModule):
       (fc): Sequential(...)
     )
     """
+
     def __init__(
-            self,
-            dl_path: Union[str, Path],
-            backbone: str = 'resnet50',
-            train_bn: bool = True,
-            milestones: tuple = (5, 10),
-            batch_size: int = 8,
-            lr: float = 1e-2,
-            lr_scheduler_gamma: float = 1e-1,
-            num_workers: int = 6,
-            **kwargs,
+        self,
+        dl_path: Union[str, Path],
+        backbone: str = "resnet50",
+        train_bn: bool = True,
+        milestones: tuple = (5, 10),
+        batch_size: int = 8,
+        lr: float = 1e-2,
+        lr_scheduler_gamma: float = 1e-1,
+        num_workers: int = 6,
+        **kwargs,
     ) -> None:
         """
         Args:
             dl_path: Path where the data will be downloaded
         """
-        super().__init__(**kwargs)
```

Comment:
that's a bit confusing. the lightning module does not accept any arguments, and there is no other superclass

Reply:
I agree, It was odd to see it there and was the cause of an error that made the training fail as a whole; Since it passed an unused argument. The `kwargs` have been removed as part of this PR